spirv_cross.hpp

```diff
@@ -94,6 +94,7 @@ struct ShaderResources
 	SmallVector<Resource> sampled_images;
 	SmallVector<Resource> atomic_counters;
 	SmallVector<Resource> acceleration_structures;
+	SmallVector<Resource> uniforms;
```

Comment:
This need to be called something more descriptive to make sure users are not confused thinking this is relevant for anything outside legacy GL. Plain uniforms are not supported in Vulkan for example. gl_plain_uniforms would be better perhaps.

Reply:
Done.